File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.py
class Solution:
    def productExceptSelf(self, nums: List[int]) -> List[int]:
        h = dict(Counter(nums))
        try:
            if h[0] > 1:
                return [0] * len(nums)
        except:
            pass

        r = []
        n = len(nums) 
        for i in range(0, n):
            if nums[i] == 0:
                r.append(int(math.prod(nums[0:i])*math.prod(nums[i+1:n])))
            else:
                r.append(int(math.prod(nums)/nums[i]))
        return r

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int max = 0;
        int c = 0;
        for(int num: nums){
            if (num == 1) {
                c++;
            }
            else {
                max = Math.max(max, c);
                c = 0;
            }
        }
        max = Math.max(max, c);
        return max;
    }
}